src/Gamecure.Core/Configuration/ReadApplicationConfigFile.cs

```diff
@@ -1,10 +1,10 @@
-﻿using Gamecure.Core.Common;
+using Gamecure.Core.Common;
 using Gamecure.Core.Common.Logging;
 using Gamecure.Core.Pipeline;
 
 namespace Gamecure.Core.Configuration;
 
-public record JiraConfig(string Url, int ProjectId, int IssueType, int Priority);
+public record JiraConfig(bool Enabled, string Url, int ProjectId, int IssueType, int Priority);
 public record PlasticConfig(string Repository, string[] Includes, string[] Excludes);
 public record LongtailConfig(string Version);
 public record AppConfig(string ClientId, string Project, string Scope, string TokenUrl, string AuthUrl, string GitCommit, string Container, PlasticConfig Plastic, LongtailConfig Longtail, JiraConfig Jira);
```

Comment:
Remove `bool Enabled`
Replace `JiraConfig Jira` with `JiraConfig? Jira`

Reply:
Btw, you can use the suggest changes markdown (pressing the button circled in the below image for a helper) to make your suggestions "acceptable". For example, it would look like:

```suggestion
public record JiraConfig(string Url, int ProjectId, int IssueType, int Priority);
public record PlasticConfig(string Repository, string[] Includes, string[] Excludes);
public record LongtailConfig(string Version);
public record AppConfig(string ClientId, string Project, string Scope, string TokenUrl, string AuthUrl, string GitCommit, string Container, PlasticConfig Plastic, LongtailConfig Longtail, JiraConfig? Jira);
```